packages/abi-typegen/README.md

```diff
@@ -0,0 +1,113 @@
+# `@fuel-ts/abi-typegen`
+
+Generate TypeScript bindings for [Sway](https://github.com/fuellabs/sway) smart contracts.
+
+See the full ABI-spec [here](https://github.com/FuelLabs/fuel-specs/blob/master/src/protocol/abi/json_abi_format.md).
+
+# Table of contents
+
+- [Documentation](#documentation)
+- [Usage](#usage)
+  - [Installation](#installation)
+  - [Full SDK Installation](#full-sdk-installation)
+- [Type's Conversion Table](#types-conversion-table)
+- [Contributing](#contributing)
+- [Changelog](#changelog)
+- [License](#license)
+
+## Documentation
+
+See [Fuel-ts Documentation](https://fuellabs.github.io/fuels-ts/packages/fuel-ts-abi-typegen/)
+
+## Usage
+
+### Installation
+
+```sh
+yarn add @fuel-ts/abi-typegen
+# or
+npm add @fuel-ts/abi-typegen
+```
+
+### Generating types
+
+When using the package in a standalone fashion, its bin is prefixed with `fuels-`.
+
+```sh
+npx fuels-typegen -i ./out/debug/*-abi.json -o ./src/contracts
+```
+
+### Full SDK Installation
+
+Alternatively, we recommend you install the [complete SDK](https://github.com/FuelLabs/fuels-ts) using the umbrella package:
+
+```sh
+yarn add fuels
+# or
+npm add fuels
+```
+
+### Generating types
+
+Note that in this example we will interact with the `fuels` bin directly.
+
+We just need to call it with the `typegen` command, and the rest feels the same.
+
+```sh
+npx fuels typegen -i ./out/debug/*-abi.json -o ./src/contracts
+```
+
+## Programatic API
+
+```ts
+import { runTypegen } from "@fuel-ts/abi-typegen";
+// OR
+// import { runTypegen } from 'fuels'
```

Comment:
I think this is not possible once the `@fuel-ts/abi-typegen` is not exported on `fuels` anymore.

Reply:
So true! 🦅